app/Http/Controllers/MapController.php

```diff
@@ -121,7 +121,7 @@ public function getCountries ()
         }
 
         /** GLOBAL LITTER MAPPERS */
-	    $users = User::where('xp', '>', 9000)
+        $users = User::where('xp', '>', -1)
```

Comment:
We can just remove the xp condition entirely, given the number of users is low, this query is really fast, so no need to limit by xp. So `User::orderBy('xp', 'desc')->...` should be ok.